src/client/src/services/connectionStatusService.ts

```diff
@@ -1,9 +1,12 @@
 import { Observable } from 'rxjs/index'
 import { map, publishBehavior, refCount } from 'rxjs/operators'
-import { Connection } from '../system'
-import { ConnectionEventType } from '../system/service/ConnectionFactory'
-import { ConnectionStatus } from '../system/service/connectionStatus'
-import { ConnectionType } from './../system/service/connectionType'
+import {
+  ConnectionEventType,
+  ConnectionStatus,
+  ConnectionType,
+  ServiceStub
+} from '../system'
+import { ConnectionEvent } from '../system'
```

Comment:
Is there a reason for not combining these separate imports from '../system' into one statement?